dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/AuthenticationStateHelper.cs

```diff
@@ -579,5 +512,128 @@ public ConfigureTrn(Configure configure)
 
                 s.OnHasIttProviderSet(hasIttProvider: ittProviderName is not null, ittProviderName);
             };
+
+        public Func<AuthenticationState, Task> TrnLookupCompletedForNewTrn(
```

Comment:
Moved all these into the `Trn` inner class since they set up state that lines up with the (Legacy) TRN journey specifically.